Description: Fix warnings in VCS_INFO's stgit support when guilt is used
 Without this patch, a git repository with guilt based patches would
 cause VCS_INFO to emit the following warning:
 "VCS_INFO_get_data_git:134: no such file or directory:
 .git/patches/master/applied"
Author: Frank Terbeck <ft@bewatermyfriend.org>

Index: zsh/Functions/VCS_Info/Backends/VCS_INFO_get_data_git
===================================================================
--- zsh.orig/Functions/VCS_Info/Backends/VCS_INFO_get_data_git	2013-01-05 04:47:29.000000000 +0100
+++ zsh/Functions/VCS_Info/Backends/VCS_INFO_get_data_git	2013-06-17 17:44:18.000000000 +0200
@@ -128,7 +128,9 @@
 rrn=${gitbase:t}
 
 local patchdir=${gitdir}/patches/${gitbranch}
-if [[ -d $patchdir ]] ; then
+if [[ -d $patchdir ]] && [[ -f $patchdir/applied ]] \
+   && [[ -f $patchdir/unapplied ]]
+then
     local -a stgit_applied stgit_unapplied stgit_all
 
     stgit_applied=(${(f)"$(< "${patchdir}/applied")"})
